libr/core/vmarks.c

```diff
@@ -14,16 +14,19 @@ R_API void r_core_visual_mark_reset(RCore *core) {
 	}
 }
 
-R_API void r_core_visual_mark_dump(RCore *core) {
+R_API bool r_core_visual_mark_dump(RCore *core) {
 	int i;
+	bool out = false;
 	if (!marks_init) {
-		return;
+		return out;
 	}
 	for (i = 0; i < UT8_MAX; i++) {
 		if (marks[i] != UT64_MAX) {
-			r_cons_printf ("fV %d 0x%"PFMT64x"\n", i, marks[i]);
+			r_cons_printf (R_CONS_CLEAR_LINE"fV %c 0x%"PFMT64x"\n", i, marks[i]);
```

Comment:
ah travis is angry because of this change. I changed the output to %c from %d because I wanted to use the API to tell users which shortcuts are available at the moment and in that case, it doesn't make sense if it's shown like 97 instead of 'a' because I assume people don't memorize the whole ascii table.

I can fix the test if you confirm this change, or tell me what to do please @radare